src/relayserver/RelayServer.ts

```diff
@@ -209,11 +234,37 @@ export class RelayServer extends EventEmitter {
   }
 
   async validateViewCallSucceeds (method: any, req: RelayTransactionRequest|DeployTransactionRequest, maxPossibleGas: number): Promise<void> {
+    /*
+    * const workerIndex = this.getWorkerIndex(req.relayRequest.relayData.relayWorker)
+    const method = this.relayHubContract.contract.methods.relayCall(
+      acceptanceBudget, req.relayRequest, req.metadata.signature, req.metadata.approvalData, maxPossibleGas)
+    let viewRelayCallRet: { paymasterAccepted: boolean, returnValue: string }
+    try {
+      viewRelayCallRet =
+        await method.call({
+          from: this.workerAddress[workerIndex],
+          gasPrice: req.relayRequest.relayData.gasPrice,
+          gasLimit: maxPossibleGas
+        })
+    } catch (e) {
+      throw new Error(`relayCall reverted in server: ${(e as Error).message}`)
+    }
+    log.debug(`Result for view-only relay call:
+paymasterAccepted  | ${viewRelayCallRet.paymasterAccepted ? chalk.green('true') : chalk.red('false')}
+returnValue        | ${viewRelayCallRet.returnValue}
+`)
+    if (!viewRelayCallRet.paymasterAccepted) {
+      throw new Error(
+        `Paymaster rejected in server: ${decodeRevertReason(viewRelayCallRet.returnValue)} req=${JSON.stringify(req, null, 2)}`)
+    }
+    * */
```

Comment:
could we please talk about why this was necessary? can be on a call if you want

Reply:
No need, i know it's not necessary but it was in the old PR and the master code has commented code too so i just merged that instead of removing it, i will remove it since it seems to be working ok and the commented code looks like old code anyway